solo-project/src/App.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import './App.css';
+import { connect } from 'react-redux';
+import { BrowserRouter as Router, Link, Route} from 'react-router-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import ProductPage from './Components/ProductPage.js';
+import { LandingPage } from './Components/LandingPage.js';
+import { getProducts } from './actions';
+import { addToCart } from './actions';
+
+
+class App extends Component {
+
+  render() {
+    return (
+      <Router>
+        <MuiThemeProvider>
+          <div className='general-container'>
+            <div className="header">
+              <div>
+                <Link to = {'/'}>
+                  <h2>BERISE</h2>
+                </Link>
+                <Route path='/admin' render = {() => (
+                  <h1>Hello Admin!</h1>
+                )}/>
+                <h3>Be confident, be free and feel great!</h3>
+              </div>
+            </div>
+            <Route exact={true} path="/" component={LandingPage} />
+            <Route path='/products' render={() => (
+              <ProductPage products={this.props.products} />
+            )} />
+
+          </div>
+        </MuiThemeProvider>
+      </Router>
+    );
+  }
+}
+
+const mapStateToProps = (state) => ({
+  products: state

```

Comment:
Although we're ignoring this right now, you're basically getting all the state in here. Think of the `App` Component as the base of your app and where your router lives (and general components as well). It's a container component that holds other container components, so all those child components will have their own `mapStateToProps`